terraform/deployments/github/repos.yml

```diff
@@ -87,6 +87,12 @@ repos:
         - Lint SCSS / Run Stylelint
         - Test Ruby / Run RSpec
 
+  content-block-editor:
+    required_status_checks:
+      standard_contexts: *standard_security_checks
+      additional_contexts:
+        - CI
```

Comment:
This is not going to work since it's a workflow name, while it should be job name. 

I suggest adding job names in https://github.com/alphagov/whitehall/tree/main/.github/workflows to use them here to configure branch protection rules (alternatively you can get ids). For workflows that use reusable workflows, the pattern is <initial_workflow.jobs.job.[name/id]> / <reused-workflow.jobs.job.[name/id]>.

There's a lot of examples in other repos.

Reply:
Sorted now - see https://github.com/alphagov/content-block-editor/blob/4e0d7e1e8a5b01f5c641a16dfe63107082fd9c7a/.github/workflows/ci.yml#L30 / https://github.com/alphagov/content-block-editor/blob/4e0d7e1e8a5b01f5c641a16dfe63107082fd9c7a/.github/workflows/playwright.yml#L7 and https://github.com/alphagov/content-block-editor/blob/4e0d7e1e8a5b01f5c641a16dfe63107082fd9c7a/.github/workflows/ci.yml#L37 / https://github.com/alphagov/content-block-editor/blob/4e0d7e1e8a5b01f5c641a16dfe63107082fd9c7a/.github/workflows/vitest.yml#L7 👍

Reply:
Thanks! This should do the trick.